guides/hosting/infrastructure/database-cluster.md

```diff
@@ -0,0 +1,41 @@
+# Database Cluster
+
+To scale Shopware beyond the horizon, we recommend you to use a database cluster. A database cluster consists of multiple read-only servers that are managed by a single primeray instance. Shopware comes in default with an read write splitting of SQL queries. When an Insert/Update/Delete query is executed, the query is distributed to the primary server and the connection uses only the primary until the end of the connection, this will be also triggered by the ``executeStatement`` in the [DebugStack decoration](https://github.com/shopware/platform/blob/v6.4.11.1/src/Core/Profiling/Doctrine/DebugStack.php#L48).
```

Comment:
```suggestion
To scale Shopware beyond the horizon, we recommend to use a database cluster. A database cluster consists of multiple read-only servers that are managed by a single primary instance. Shopware comes in default with an read write splitting of SQL queries. When an [`INSERT`/`UPDATE`/`DELETE`/...](https://github.com/shopware/platform/blob/v6.4.11.1/src/Core/Profiling/Doctrine/DebugStack.php#L48) query is executed, the query is distributed to the primary server and the connection uses only the primary node until the end of the connection. This will be ensured by the ``executeStatement`` in the [DebugStack decoration](https://github.com/shopware/platform/blob/v6.4.11.1/src/Core/Profiling/Doctrine/DebugStack.php#L48).
```